### PR TITLE
ACT-2883 | Updated dependecies

### PR DESCRIPTION
**Pull request by avolinsky**

### CODE REVIEW

File: src/sdk/resources/personalDrive.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { IHttpClient } from '../http';
 import { PagedResource } from './resource';
 import { PersonalDriveMatchesResource } from './personalDriveMatches';
+import { Account } from '..';
```
Comment (gotvino) [inline src/sdk/resources/personalDrive.ts:4]: Why was this needed?

Comment (avolinsky) [reply src/sdk/resources/personalDrive.ts]: `Account` was part of global API until recent TypeScript update..

**Review by gotvino — APPROVED** [review]
